release/xgboost_tests/workloads/train_gpu.py

```diff
@@ -26,7 +26,7 @@
 import ray
 from xgboost_ray import RayParams
 
-from ray.util.xgboost.release_test_util import train_ray
+from release_test_util import train_ray
```

Comment:
This will presumably not work because the remote workers won't be able to find the relative imports - we will have to include them in the runtime env